aten/src/ATen/NestedTensorImpl.cpp

```diff
@@ -172,6 +173,7 @@ NestedTensorImpl::NestedTensorImpl(
       nested_stride_tensor_(std::move(nested_stride_tensor)),
       storage_offsets_(std::move(offsets)),
       opt_sizes_(construct_opt_sizes(nested_size_tensor_)) {
+  C10_LOG_API_USAGE_ONCE("torch.nested_tensor");
```

Comment:
nit: torch.nested.nested_tensor?

Reply:
This is just the line that gets logged so I figured we would just want a phrase that semi-uniquely indicates nested tensors were used

Reply:
Then why not write "Using NestedTensor"?